programs/drift/src/state/perp_market.rs

```diff
@@ -1143,6 +1143,35 @@ impl AMM {
 
         Ok(())
     }
+
+    pub fn update_last_oracle_conf_pct(
```

Comment:
nit: think makes it sound like the function would mutate the amm, but it just calculates the new value. should maybe be get_new_oracle_conf_pct?